### DISCUSSION TITLE
Is Sharpmap suitable for the implementation of a WMS server ?

### DISCUSSION THREAD

**Discussion by nicoraffin** [SharpMap/SharpMap#211] [Q&A] [answered]
Hi !

I have to expose geospacial data from an ORACLE database. 
Is Sharpmap suitable for the implementation of a WMS server ? 
Is there a better solution for .net developers ?

Thank you for your answers :)

**Comment by DGuidi** [comment, marked as answer]
> Is there a better solution for .net developers ?

If you are stick to open source implementations, the only library I know that is dotnet native is sharpmap. otherwise probably I will take a look at ArcGIS libraries and/or software, or google for other kind of solutions.
You can build a sample quickly to see if SharpMap matches your needs: the only doubt I have (not experiences from my side) is the hardness to configure Oracle native libraries, actually :)
Anyway, if you need a WMS server to expose, maybe you can think also about using other non-.net solutions like GeoServer and MapServer.
